### DISCUSSION TITLE
Is it possible to use BLE on esp32-solo ?

### DISCUSSION THREAD

**Discussion by ramafun** [arendst/Tasmota#13813] [Q&A] [answered]
I would need that to use a Bluetooth remote with Shelly 1 Plus

**Comment by sfromis** [comment, marked as answer]
Since there is a tasmota32solo1-bluetooth.bin build among the "[unofficial](https://github.com/tasmota/install/tree/main/firmware/unofficial)" binaries, the answer is not no. Not sure if it is as stable or not. (Not that I have reasons to assume "not").

**Reply by ramafun** [reply]
Thank you very much for this hint! I just tried out and it works seemingly.
